kodipydent/KodiClient.py

```diff
@@ -57,7 +73,7 @@ def random_id_gen():
     return uuid.uuid4().hex
 
 def Kodi(hostname, username='kodi', password=None, port=8080):
-    get_rpc = API(base_hive, hostname=hostname, port=port, username=username, password=password)
+    get_rpc = API(base_hive, hostname=hostname, port=port, username=username, password=password, request_id=random_id_gen())
```

Comment:
Two changes to this line, please.

1. Let's switch to a hanging indent, as below for the returned API object - this change takes us over the 120-character single-line limit.
2. Instead of `request_id=random_id_gen()`, do `request_id=random_id_gen`. When we pass a callable, it'll be used to generate a unique ID for reach request; what we've currently got here will instead create a single value that'd be used for every single request (which, since it's a single request, should be OK, but still isn't ideal).